src/main/java/com/google/firebase/auth/AbstractFirebaseAuth.java

```diff
@@ -343,8 +345,11 @@ public FirebaseToken verifySessionCookie(String cookie) throws FirebaseAuthExcep
    * checkRevoked} is true, throws a {@link FirebaseAuthException}.
    *
    * @param cookie A Firebase session cookie string to verify and parse.
-   * @param checkRevoked A boolean indicating whether to check if the cookie was explicitly revoked.
+   * @param checkRevoked A boolean indicating whether to check if the cookie was explicitly revoked
```

Comment:
Same question/comment here as for token revoked.  Maybe "if the user is disabled" doesn't belong in either boolean's description?

Reply:
I think it belongs here. I'm not quite sure what you mean, maybe my comment above clarifies things?